.storybook/local-preset.js

```diff
@@ -8,3 +11,44 @@ function managerEntries(entry = []) {
 module.exports = {
   managerEntries,
 };
+
+function serve() {
+  // Temporarily create a server as a psuedo server channel. This is throwaway code
+  const app = express();
+
+  app.post("/build", async (req, res) => {
+    let sent = false;
+    res.setHeader("access-control-allow-origin", "*");
+
+    await run({
+      flags: {
+        projectToken: "00baf09dbbe8",
```

Comment:
I imagine we'd want to ask the user to put this in their `main.ts` file?

maybe:
```ts
const { projectToken } = await options.presets.apply('chromatic')
```

Reply:
@ndelangen sure, but where do you get a reference to `options` from?

Reply:
Any preset value can access the options object as a second argument

Reply:
@tmeasday you can gain access like so:

Because you're not really interested in hooking into/modifying anything, you can use any presetValue you know storybook will call, like `stories`, `core`, `addons` and return the value unmodified.

```ts
export default {
  addons: async (current, options) => {
    // now you have access to options
    // const out = await options.presets.apply('whatever', 'default-value')
    
    // return unmodified
    return current;
  }
}
```